cmd/routedns/config.go

```diff
@@ -92,6 +92,7 @@ type cacheBackend struct {
 	RedisMaxRetries      int    `toml:"redis-max-retries"`       // Maximum number of retries before giving up. Default is 3 retries; -1 (not 0) disables retries.
 	RedisMinRetryBackoff int    `toml:"redis-min-retry-backoff"` // Minimum back-off between each retry. Default is 8 milliseconds; -1 disables back-off.
 	RedisMaxRetryBackoff int    `toml:"redis-max-retry-backoff"` // Maximum back-off between each retry. Default is 512 milliseconds; -1 disables back-off.
+	RedisAsyncSetOnMiss  *bool  `toml:"redis-async-set-on-miss"` // When true (default), perform Redis SET asynchronously on cache miss to reduce latency.
```

Comment:
I think we should invert this to have a default of false which is more aligned with the rest, something where false == write asynchronously. Also, the `OnMiss` in the name is confusing. Maybe just `RedisSyncSet`?